extensions/arc/notebooks/arcDeployment/deploy.sql.existing.arc.ipynb

```diff
@@ -99,6 +99,7 @@
                 "print (f'Creating the SQL managed instance - Azure Arc instance')\n",
                 "\n",
                 "is_indirect = arc_data_controller_connection_mode == 'indirect'\n",
+                "is_general_purpose = sql_service_tier == 'GeneralPurpose'\n",
```

Comment:
Just to verify - you don't have to do any of the case-insensitive stuff you did for the connection mode?